inference/core/version.py

```diff
@@ -1,4 +1,43 @@
+import requests
+import time
+import threading
+from inference.core.logger import logger
+
 __version__ = "0.8.5"
 
+latest_release = None
```

Comment:
All this code should probably live in a separate file, `check_version.py`, maybe in a utils folder? Then we can choose more easily when to print the warning. How often do you think we should print the message, every 5 minutes, on every infer?

What do you think @paulguerrie ?

Reply:
I could be convinced either way. I think this is a fine spot for it.

Reply:
Ok, lets 🚢 it. We can update later as needed.